docs/Permissions/Onchain-Permissioning.md

```diff
@@ -0,0 +1,213 @@
+description: Onchain Permissioning
+<!--- END of page meta data -->
+
+# Onchain Permissioning 
+
+Onchain permissioning uses smart contracts to store and maintain the node whitelist. Using onchain permissioning
+enables all nodes to read the node whitelist from one location. 
+                                                                       
+!!! note
+    Pantheon supports onchain permissioning for nodes. Onchain permissioning for accounts will be available 
+    in a future Pantheon release. 
+
+We have provided a set of smart contracts that interface with onchain permissioning in the 
+[PegaSysEng/permissioning-smart-contracts](https://github.com/PegaSysEng/permissioning-smart-contracts) repository. 
+We have also provided a management interface to interact with the contracts. 
+
+## Provided Contracts
+
+The following smart contracts are provided in the [PegaSysEng/permissioning-smart-contracts](https://github.com/PegaSysEng/permissioning-smart-contracts) repository: 
+
+* Ingress - a simple contract functioning as a gateway to the Admin and Rules contracts. The Ingress contract is deployed 
+to a static address. 
+
+* Rules - stores the node whitelist. 
+
+* Admin - stores the list of admin accounts.
+
+## Pre-requisites 
+
+For nodes that are going to maintain the nodes whitelist or the list of admin accounts: 
```

Comment:
Not sure if it's mentioned elsewhere - there's a few implicit configuration prerequisites: JSON-RPC enabled + appropriate methods etc.

Reply:
Following this up separately and will cover in a new PR if needed